docs/DeveloperGuide.md

```diff
@@ -80,31 +80,56 @@ The following sequence diagram shows how the Priority operation works:
   * Pros: Will use less memory (No need to show the parser field).
   * Cons: Increased coupling
 
-### \[Proposed\] Policy feature
+### Policy feature
 
-#### Proposed Implementation
+#### Implementation
 
-Policy (class) is a field in Person that is uniquely different from the current fields in Person
- such as phone, address, email, etc. A Person can have up to a single Policy. 
- 
- Policy class contains 3 attributes: String name, String description.
+Policy class is implemented as shown in this class diagram:
```

Comment:
Perhaps we should use `Policy` to format the name, instead of Policy
Same for other objects, classes, variables in this DG e.g. `Person`, `PolicyName`

Reply:
Sure. Then we need to make this consistent throughout the DG too!